content/en/blog/_posts/2023-08-21-job-update-post.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Improved failure handling for Jobs"
+date: 2023-08-21
+slug: kubernetes-1-28-jobapi-update
+---
+
+**Authors:** Kevin Hannon (G-Research), Michał Woźniak (Google)
+
+This blog discusses two new features in Kubernetes 1.28 to improve Jobs for batch
+users: [Pod replacement policy](/docs/concepts/workloads/controllers/job/#pod-replacement-policy)
+and [Backoff limit per index](/docs/concepts/workloads/controllers/job/#backoff-limit-per-index).
+
+These features provide improvements to the handling of pod failures in Job, which
+continue the effort started by the
+[Pod failure policy](/docs/concepts/workloads/controllers/job/#pod-failure-policy)
+feature.
+
+## Pod replacement policy
+
+### What problem does this solve?
```

Comment:
I would just remove this line so that we don't have subtitles (line 19) without a paragraph.

Reply:
I suppose you meant line 21. If so, sgtm. I think "Pod replacement policy" and "Backoff limit per index" are useful to start the sections.

Reply:
yes, just line 21

Reply:
Done